File: 0217-contains-duplicate/0217-contains-duplicate.cpp
class Solution {
public:
    bool containsDuplicate(vector<int>& nums) {
        unordered_map<int, int> mp;
        for(auto it: nums){
            if(mp.count(it))
                return true;
            mp[it]++;
        }
        return false;
    }
};

File: 0217-contains-duplicate/0217-contains-duplicate.kt
class Solution {
    fun containsDuplicate(nums: IntArray): Boolean {
        val set = HashMap<Int, Int>()
        for (i in 0 until nums.size) {
            if (set.containsKey(nums[i])) return true
            set.put(nums[i], 1)
        }
        return false
    }
}